src/figuresClasses.ts

```diff
@@ -1,19 +1,93 @@
+enum FigureShape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
 export interface Figure {
+  shape: FigureShape.Triangle | FigureShape.Circle | FigureShape.Rectangle;
+  color: string;
```

Comment:
not just any string 
![image](https://github.com/mate-academy/ts_figures_classes/assets/97547824/bed9befc-8ea0-4b29-8933-958d860f8f17)